components/Navbar/NavbarMobile/Menu.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import TabGroupMobile from 'components/Navbar/NavbarMobile/TabGroupMobile'
+import clsx from 'clsx'
+
+export default function Menu(props) {
+  return (
+    <div
+      id="lesson-navigation-menu"
+      className={clsx('absolute z-10 w-full bg-back duration-[400ms]', {
+        'h-full': props.isOpen,
+        'h-0 delay-200': !props.isOpen,
+      })}
+      style={{}}
+      aria-modal="true"
+      aria-hidden={!props.isOpen}
+      aria-label="Lesson Navigation Menu"
+    >
+      {
```

Comment:
Why is `TabGroupMobile` inside an curly braces here? Did our formatter do that?

Reply:
I also noticed them earlier but didn't touch them because I thought they might be here for some reason.

Turns out they weren't :)

Addressed in 9b4c5743cb1375d2eae011ab956848b2f92682c6